docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
The docs folder is where the ReadTheDocs pages (in ReST format) are found.

Reply:
We can edit all these files in docs/* later.

Reply:
I noticed that the LaTeX documentation has been moved from the doc/
directory into the docs/ directory. That's fine but now it is difficult
to see which files belong to the LaTeX User Manual and which to
ReadTheDocs. Maybe we can separate them into two subfolders? For
example:

docs/UserManual contains:

```
kpp_UserManual.tex
kpp_UserManual.pdf
rolf.bib
adrian.bib
Figures/*
```

docs/ReadTheDocs contains:

```
make.bat
Makefile
requirements.txt
source/*
```

Also, we need to change doc/ to docs/ in .gitignore

Reply:
Upon further review, it seeems ReadTheDocs [assumes a certain folder structure](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/folders.html).  

Maybe a better solution is to put the PDF doc into a folder such as `pdf_manual` off of the root folder.

Reply:
Yes. Maybe just `manual` instead of `pdf_manual`?

At some point we should probably discuss how we continue with the two different documentations. Merge them? Or use the LaTeX manual for a generic description and ReadTheDocs for geos-chem specific instructions?

Reply:
For now I can make it `manual`.

I think most open-source projects now have associated ReadTheDocs documentation because you can peg the documentation files to specific releases.  I think eventually it'd make sense to move entirely to RTD but that is a while down the road.

Reply:
I don't mind moving to ReadTheDocs but I guess it will be quite a bit of
work to convert the current User Manual into the *.rst format. Well, we
can discuss this later...

Reply:
We can potentially use [pandoc](https://pandoc.org/) to convert the manual to rst.

Reply:
Thanks, @laestrada! Yes, maybe we can use pandoc. I made a quick test
with

`pandoc -o kpp_UserManual.rst kpp_UserManual.tex`

At first glance, the main text looks fine. However, I fear that the
tables need special attention.

Reply:
There is always a little manual manipulation that is needed w/ Pandoc.  But I think it's not too hard to update those.